mirgecom/profiling.py

```diff
@@ -51,15 +52,68 @@ class SingleCallKernelProfile:
     footprint_bytes: int
 
 
+class StatisticsAccumulator:
+    """Class that provides statistical functions for profile results.
+
+    Values are converted to "Giga".
```

Comment:
Why not introduce a `scale_factor` constructor parameter (defaulting to 1) so as to not wreck the general-purpose-ness?

Reply:
Done.

Reply:
Doesn't look done. Forgot to push?